### ISSUE TITLE
wrong data in month-output; shifted 1 day

### ISSUE THREAD

**Issue by SBFspot** [SBFspot/SBFspot#60]
When using the command:  ./SBFspot -v -am12   the output is not correct. the data of yesterday seems not present, but is shown with the date of the day before yesterday, and so on.

#### This work item was migrated from CodePlex
CodePlex work item ID: '54'
Assigned to: 'SBF'
Vote count: '1'

**Comment by SBFspot** [comment]
[hvdkamer@10/09/2014]
I see the same with normal logging:

```
dd-MM-yyyy;kWh;kWh
01-09-2014;365,561;5,360
02-09-2014;375,804;10,243
03-09-2014;387,855;12,051
04-09-2014;395,023;7,168
05-09-2014;401,345;6,322
06-09-2014;409,893;8,548
07-09-2014;420,559;10,666
08-09-2014;425,074;4,515
```

The line for 08-09 is for yesterday (09-09). At the end of that day the total produced was 425,074 and for the day self it was 4,515. The results for 01-09 are missing, the shown values are for 02-09.

**Comment by SBFspot** [comment]
**Issue closed by _SBF_ with comment**
Fixed in V3.0.1

**Reason closed**
Fixed
